syntax = "proto3";

package rules;

message AccelRequest {
    string symbol = 1;
    int32 lfast = 2;
    bool use_attenuation = 3;
    float scaling_factor = 4;
    string scaling_type = 5;

}

message AccelResponse {
    bytes series = 1;
}

service Accel{
    rpc get_accel (AccelRequest) returns (AccelResponse);
}